src/hssm/hssm.py

```diff
@@ -230,15 +242,29 @@ def __init__(
                         + "`model_config`."
                     )
                 self.model_config = model_config
-                self.list_params = model_config["list_params"]
+                self.list_params = model_config["list_params"][:]
             else:
                 # For supported models without configs,
                 # We don't require a model_config (because list_params is known)
                 model = cast(SupportedModels, model)
                 self.model_config = {} if model_config is None else model_config
-                self.list_params = default_params[model]
+                self.list_params = default_params[model][:]
 
         self._parent = self.list_params[0]
+        if p_outlier is not None and lapse is None:
+            raise ValueError(
+                "You have specified `p_outlier`. Please also specify `lapse`."
+            )
+        if lapse is not None and p_outlier is None:
+            raise ValueError(
+                "You have specified `lapse`. Please also specify `p_outlier`."
+            )
+        if "p_outlier" in self.list_params:
```

Comment:
btw. just flagging for future versions. 

Rn is it possible to specify the `p_outlier` via a regression model and make it hierarchical?

Reply:
Not right now but that's why I wanted this to be reviewed last. I thought of a few things to do this more consistently